msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,619 @@
+- Integrate with external services including CRM and project management.
+- Add custom ribbon buttons or menu items to perform specific tasks.
+
+Combining an Outlook Add-in with a Teams app enables scenarios that can't be satisfied with either of them separately. For example, consider an app that enables sales persons to insert a customized discount into an email for a customer, and then records the offer in a backend database. The app also enables the sales manager to view data about all the discounts that have been offered in a Teams tab. There is a sample of this app with animated screen shots at [Discount Offers](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-add-in-combined/nodejs).
+
+This article will walk you through the steps to add an Outlook Add-in to a Teams app, test it, then deploy it to Azure.
+
+## Prerequisites
+
+To configure an Office Add-in as additional capability, you must meet the following conditions:
+
+- You have a Microsoft 365 account that includes Teams, or separate subscriptions to both Microsoft 365 and Teams, to test the application. For example, an *.onmicrosoft.com test account through the [Microsoft 365 Developer Program](/office/developer-program/microsoft-365-developer-program).
+- Your Microsoft 365 account has been added as an account in desktop Outlook. See [Add an email account to Outlook](https://support.microsoft.com/office/add-an-email-account-to-outlook-e9da47c4-9b89-4b49-b945-a204aeea6726)
+- To deploy the Teams app to Azure as described in the last section of this article, you need an Azure account with an active subscription. Create your free Azure account if you don't already have one by using the link [Free Azure account](https://azure.microsoft.com/free/).
+- A Teams app that was created with the latest version of Teams Toolkit.
+
+## Overview
+
+The following are the major steps to adding an Outlook Add-in to a Teams app.
+
+1. [Prepare the Teams app project](#prepare-the-teams-app-project)
+1. [Create an Office Add-in project](#create-an-outlook-add-in-project) that is initially separate from your Teams app project.
+1. [Merge the manifest](#merge-the-manifest) from the Outlook Add-in project into the unified manifest for Microsoft 365.
+1. [Copy the Outlook Add-in files to the Teams app project](#copy-the-outlook-add-in-files-to-the-teams-app-project).
+1. [Edit the tooling configuration files](#edit-the-tooling-configuration-files).
+1. [Run the app and add-in locally at the same time](#run-the-app-and-add-in-locally-at-the-same-time)
+1. [Move the application to Azure](#move-the-application-to-azure).
+
+## Prepare the Teams app project
+
+Begin by separating the source code for the tab (or bot) into its own subfolder. These instructions assume that the project initially has the following structure. To create a Teams app project with this structure, be sure you're using the latest released version of Teams Toolkit. 
```

Comment:
```suggestion
Create a Teams tab or bot using the latest version on Teams Toolkit.  After the project is created, you'll see the following  project directories and files under **EXPLORER** in Visual Studio Code.
```

Reply:
This changes the meaning of the paragraph. They do NOT have to create a Teams app. They just have to have one that matches the structure that is displayed below this paragraph. One way, but not the only way, to get one is to create it with the latest version of TTK. Also, the audience of this article does not need to be told how to view a folder structure.